schema/bls.mcf

```diff
@@ -15337,18 +15337,86 @@ rangeIncludes: schema:Number
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember1983Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1983 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember1988Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1988 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember1982Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1982 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriod1967Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is 1967 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:InexPointBasePeriod1982To1984Equals100
 typeOf: dcs:UnitOfMeasure
 description: "The reference base is 1982 to 1984 equals 100."
 descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
 
+Node: dcid:IndexPointBasePeriodDecember1986Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1986 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember1990Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1990 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember2005Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 2005 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember1997Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1997 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember1996Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1996 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember1984Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 1984 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember2007Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 2007 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
+Node: dcid:IndexPointBasePeriodDecember2009Equals100
+typeOf: dcs:UnitOfMeasure
+description: "The reference base is December 2009 equals 100."
+descriptionUrl: "https://www.bls.gov/cpi/technical-notes/home.htm"
+
 Node: dcid:IndexPointBasePeriodDecember1999Equals100
-name: "IndexPointBasePeriodDecember1999Equals100"
```

Comment:
Do you think there exists a human readable name that might concisely hint at what this means without looking at the description? If so we might be able to use it in charts, etc.

Reply:
Maybe ReferenceBaseDecember1999Equals100 and get rid of the description?

Reply:
Hmm.. slightly prefer current over that proposal. WBU?

Reply:
Yeah I guess I don't really see the problem with the current name. If we want the name to be short, at least "December1999Equals100" needs to be in the name.

Reply:
Just to be clear--Currently there is no name (just dcid).

Reply:
I meant in the identifier. What are the advantages of having a name? I thought the dcid would still be displayed so we really just needed a descriptive dcid

Reply:
Names are useful when the dcids are not human readable. This is a borderline case, but thanks for reminding me--the enums in this PR should get names.